### ISSUE TITLE
[IMPROVEMENT] Have backup progress icon on Backup page

### ISSUE THREAD

**Issue by jenting** [longhorn/longhorn#3168]
**Is your feature request related to a problem? Please describe.**

After the #3035 be enhanced, the backend endpoint (RESTful/Websocket) will have a progress and error message. UI could
- add a progress bar to let users know the backup creation progress just like the progress bar when creating backup shown on the `Snapshots and Backups` under Volume page.
- show the pop-out tip when the Backup state `Error` and do not display `Invalid date` on `Snapshot Created` column.
![Screenshot from 2021-11-02 11-08-38](https://user-images.githubusercontent.com/49380831/139780744-f3afdd6e-208d-4318-999b-98415d770cb6.png)
- consider displaying a different color like `red` when the backup creation is in `Error` state. (or keep it as-is cc @longhorn/qa, WDYT?)
![Screenshot from 2021-11-02 11-10-10](https://user-images.githubusercontent.com/49380831/139780860-15f1ce3a-95e9-4a71-91ae-aae166e560ec.png)

**Describe the solution you'd like**

- Have a progress bar for the in-progress backup under the Backup page.
- Shown the error reason when the Backup state `Error` and do not display `Invalid date` on `Snapshot Created` column.

**Comment by khushboo-rancher** [comment]
Validated this with master-head `5db6c8d` and v1.2.x-head `45144ea`

Validation - **Fail**

I enabled recurring backup job, I didn't see any progress on the backup, it always show 0% an then completed.

<img width="1355" alt="Screen Shot 2021-11-10 at 2 39 06 PM" src="https://user-images.githubusercontent.com/60111667/141205548-d59cd66d-37fb-4f4e-bf38-833232dd2e40.png">

**Comment by jenting** [comment]
The backend code (manager/engine) code hasn't merged yet.

**Comment by khushboo-rancher** [comment]
Thanks @jenting for the info, @smallteeths Can you help in moving this to `ready for testing` bucket once the backend code is merged.

**Comment by innobead** [comment]
@smallteeths I think we missed https://github.com/longhorn/longhorn/issues/3035 needed to merge first. Could you help revert those PRs and have draft PRs to merge until https://github.com/longhorn/longhorn/issues/3035 is ready? Thanks.

**Comment by smallteeths** [comment]
> @smallteeths I think we missed #3035 needed to merge first. Could you help revert those PRs and have draft PRs to merge until #3035 is ready? Thanks.

ok

**Comment by jenting** [comment]
Backend code merged into the master branch.

**Comment by khushboo-rancher** [comment]
Validated on master-head and v1.2.x-head - `12/10/2021`

Validation - **Pass**

The Creation state shows the state as Progress with progress bar and percentage, completed, error.

<img width="1769" alt="Screen Shot 2021-12-10 at 12 30 33 PM" src="https://user-images.githubusercontent.com/60111667/145656513-a58f45c5-06cb-4aea-ab0b-ced344379a5e.png">
